app/assets/stylesheets/forms.scss

```diff
@@ -17,14 +17,6 @@ select.form-element {
 
   @media screen and (min-width: $screen-md) {
     flex-direction: row;
-
```

Comment:
What is the effect of removing these?

Reply:
@cellio no effect that I managed to notice while checking other than the positioning of the "delete" button being fixed. All the other instances have CSS that make these rules redundant.